src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -665,6 +665,11 @@ internal virtual bool SupportsExtensionMethods
 
             foreach (Symbol member in members)
             {
+                if ((member as FieldSymbol)?.AssociatedSymbol is EventSymbol)
+                {
+                    continue;
+                }
```

Comment:
Do you have suggestions for alternatives? This is added to avoid the assertion on line 72:

https://github.com/dotnet/roslyn/blob/374d1282b4a3e8383dcd03b0f25b6291f7f8e836/src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs#L72